experimental/class-features.md

```diff
@@ -0,0 +1,63 @@
+
+# Class Features
+
+This experimental extension covers three class features proposals: 
+[Class Fields], [Static Class Features] and [Private Methods].
+
+## ClassBody
+
+```js
+extend interface ClassBody <: Node {
+    body: [ MethodDefinition | PropertyDefinition ];
+}
+```
+
+## PropertyDefinition
+
+```js
+interface PropertyDefinition <: Node {
+    type: "PropertyDefinition";
+    key: Expression | PrivateName;
+    value: Expression;
```

Comment:
The initializer of [FieldDefinition](https://tc39.es/proposal-class-fields/#sec-new-syntax) production is optional. So should the `value` nullable?